### ISSUE TITLE
Uncaught ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by ajaypillay** [appmotion/apollo-augmented-hooks#3]
Hi, so I've been experimenting around with migrating my hooks to this package but I have this error thrown in my browser console whenever I try to execute a mutation.

I'm using a Meteor/Apollo stack so my client code is as follows:
```js
import React from "react"
import { Meteor } from "meteor/meteor"
import { render } from "react-dom"
import { BrowserRouter } from "react-router-dom"
import { ApolloClient, HttpLink, split, InMemoryCache, ApolloProvider } from "@apollo/client"
import { ApolloLink, from } from "apollo-link"

import { DDPSubscriptionLink, isSubscription } from 'apollo-link-ddp'
import { offsetLimitPagination } from "@apollo/client/utilities"
import { initAugmentedHooks } from "apollo-augmented-hooks"
import App from "../../ui/App"

const httpLink = new HttpLink({ uri: "http://XXX.XXX.XX.XX:3000/graphql" })

const authLink = new ApolloLink((operation, forward) => {
    const token = Accounts._storedLoginToken()
    operation.setContext(() => ({
        headers: {
            authorization: token
        }
    }))
    return forward(operation)
})

const subscriptionLink = new DDPSubscriptionLink()

const splitLink = split(
    isSubscription,
    subscriptionLink,
    httpLink
)

const cache = new InMemoryCache({
    typePolicies: {
        Query: {
            fields: {
                getFeed: offsetLimitPagination()
            }
        }
    }
})

const client = new ApolloClient({
    link: from([authLink, splitLink]),
    cache
})

initAugmentedHooks(client)

const ApolloApp = () => (
    <BrowserRouter>
        <ApolloProvider client={client}>
            <App client={client} />
        </ApolloProvider>
    </BrowserRouter>
)

Meteor.startup(() => {
    render(<ApolloApp />, document.getElementById("app"))
})
```
And my mutation is merely as follows:
```gql
const EDIT_THING = gql`
    mutation EditThing($someInt: Int!) {
        editThing(someInt: $someInt)
    }
`
```
Initialized as such:
```js
import { useMutation } from "apollo-augmented-hooks"
import { gql } from "@apollo/client"
const sendMutation = useMutation(EDIT_THING)
```
Further in my component I have:
```js
sendMutation({ variables: { someInt: IntFromComponent }})
```
But this throws `Uncaught ReferenceError: regeneratorRuntime is not defined` in my console for some reason. I'm using the latest release, 1.8.0. Have also tried removing the package and re-installing it to no avail.

**Comment by mindnektar** [comment]
Hey Ajay! I won't be able to fix this until Monday, but it's because I forgot to add `@babel/polyfill` as a dependency, which is required for things like `async/await`. It works for me, because the projects in which I am using `apollo-augmented-hooks` include it themselves. You can do the same thing in the meantime, until I add the dependency to this package!

**Comment by ajaypillay** [comment]
Ah I see. I shall probably wait till then and work on other things in the meantime. Thanks Martin!

**Comment by mindnektar** [comment]
I've released v1.8.1 - it should work now.
